@vadz Errors for locale stuff:

1. Native uilocale breaks down:

```
[  2%] Building OBJCXX object libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/osx/cocoa/utils_base.mm.o
cd /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base && /opt/local/bin/g++-mp-14 -DWXBUILDING -DWXDLLNAME=wx_baseu-3.3 -DWXMAKINGDLL_BASE -DWX_PRECOMP -D_FILE_OFFSET_BITS=64 -D_UNICODE -D__WXGTK3__ -D__WXGTK__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -Dwxbase_EXPORTS -I/opt/local/include -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/lib/wx/include/gtk3-unicode-3.3 -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include -pipe -Os -F/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks -U__APPLE_ALTIVEC__ -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -mmacosx-version-min=10.6 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wundef -Wunused-parameter -Wno-shorten-64-to-32 -Wno-ctor-dtor-privacy -Woverloaded-virtual -Winvalid-pch -include /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/secretstore.cpp.o -MF CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/secretstore.cpp.o.d -o CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/secretstore.cpp.o -c /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/secretstore.cpp
cd /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base && /opt/local/bin/g++-mp-14 -DWXBUILDING -DWXDLLNAME=wx_baseu-3.3 -DWXMAKINGDLL_BASE -DWX_PRECOMP -D_FILE_OFFSET_BITS=64 -D_UNICODE -D__WXGTK3__ -D__WXGTK__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -Dwxbase_EXPORTS -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/lib/wx/include/gtk3-unicode-3.3 -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include -x objective-c++ -pipe -Os -F/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks -U__APPLE_ALTIVEC__ -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/include -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -mmacosx-version-min=10.6 -fPIC -fvisibility-inlines-hidden -Wall -Wundef -Wunused-parameter -Wno-shorten-64-to-32 -Winvalid-pch -include /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base/CMakeFiles/wxbase.dir/cmake_pch.objcxx.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/osx/cocoa/utils_base.mm.o -MF CMakeFiles/wxbase.dir/__/__/__/__/src/osx/cocoa/utils_base.mm.o.d -o CMakeFiles/wxbase.dir/__/__/__/__/src/osx/cocoa/utils_base.mm.o -c /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm: In function 'NSLocale* wxGetCurrentNSLocale()':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:47:98: warning: 'NSLocale' may not respond to '+localeWithLocaleIdentifier:'
   47 |         static wxCFRef<NSLocale*> stdCLocale([[NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"] retain]);
      |                                                                                                  ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:47:98: warning: (messages without a matching method signature will be assumed to return 'id' and accept '...' as arguments)
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm: In static member function 'static {anonymous}::wxUILocaleImplCF* {anonymous}::wxUILocaleImplCF::Create(const wxLocaleIdent&)':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:26: error: expected ';' before 'in'
  107 |         for ( id nsLocId in [NSLocale availableLocaleIdentifiers] )
      |                          ^~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:26: error: 'in' was not declared in this scope; did you mean 'yn'?
  107 |         for ( id nsLocId in [NSLocale availableLocaleIdentifiers] )
      |                          ^~
      |                          yn
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:39: error: expected primary-expression before 'availableLocaleIdentifiers'
  107 |         for ( id nsLocId in [NSLocale availableLocaleIdentifiers] )
      |                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:39: error: expected ']' before 'availableLocaleIdentifiers'
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:39: error: expected ';' before 'availableLocaleIdentifiers'
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/utilsunx.cpp:227:32: warning: "MAC_OS_X_MIN_VERSION_REQUIRED" is not defined, evaluates to 0 [-Wundef]
  227 | #elif (defined(__DARWIN__) && (MAC_OS_X_MIN_VERSION_REQUIRED >= 1090)) || defined(__STDC_LIB_EXT1__)
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:39: error: 'availableLocaleIdentifiers' was not declared in this scope
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:65: error: expected ')' before ']' token
  107 |         for ( id nsLocId in [NSLocale availableLocaleIdentifiers] )
      |                                                                 ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:13: note: to match this '('
  107 |         for ( id nsLocId in [NSLocale availableLocaleIdentifiers] )
      |             ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:65: error: expected primary-expression before ']' token
  107 |         for ( id nsLocId in [NSLocale availableLocaleIdentifiers] )
      |                                                                 ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:107:18: warning: unused variable 'nsLocId' [-Wunused-variable]
  107 |         for ( id nsLocId in [NSLocale availableLocaleIdentifiers] )
      |                  ^~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:150:79: warning: 'NSLocale' may not respond to '+localeWithLocaleIdentifier:'
  150 |         auto nsloc = [NSLocale localeWithLocaleIdentifier: cfName.AsNSString()];
      |                                                                               ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm: In member function 'virtual wxString {anonymous}::wxUILocaleImplCF::GetLocalizedName(wxLocaleName, wxLocaleForm) const':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:237:92: warning: 'NSLocale' may not respond to '-localizedStringForLocaleIdentifier:'
  237 |             str = [convLocale localizedStringForLocaleIdentifier:[m_nsloc localeIdentifier]];
      |                                                                                            ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:240:83: warning: 'NSLocale' may not respond to '-languageCode'
  240 |             str = [convLocale localizedStringForLanguageCode:[m_nsloc languageCode]];
      |                                                                                   ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:240:84: warning: 'NSLocale' may not respond to '-localizedStringForLanguageCode:'
  240 |             str = [convLocale localizedStringForLanguageCode:[m_nsloc languageCode]];
      |                                                                                    ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:243:81: warning: 'NSLocale' may not respond to '-countryCode'
  243 |             str = [convLocale localizedStringForCountryCode:[m_nsloc countryCode]];
      |                                                                                 ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:243:82: warning: 'NSLocale' may not respond to '-localizedStringForCountryCode:'
  243 |             str = [convLocale localizedStringForCountryCode:[m_nsloc countryCode]];
      |                                                                                  ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm: In member function 'virtual wxLayoutDirection {anonymous}::wxUILocaleImplCF::GetLayoutDirection() const':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:344:110: warning: 'NSLocale' may not respond to '-languageCode'
  344 |     NSLocaleLanguageDirection layoutDirection = [NSLocale characterDirectionForLanguage:[m_nsloc languageCode]];
      |                                                                                                              ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm: In static member function 'static wxUILocaleImpl* wxUILocaleImpl::CreateStdC()':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:360:84: warning: 'NSLocale' may not respond to '+localeWithLocaleIdentifier:'
  360 |     return new wxUILocaleImplCF([NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"]);
      |                                                                                    ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm: In static member function 'static wxVector<wxString> wxUILocaleImpl::GetPreferredUILanguages()':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:383:52: error: no matching function for call to 'wxCFStringRef::AsString(NSArray&)'
  383 |         preferred.push_back(wxCFStringRef::AsString(preferredLangs[j]));
      |                             ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
In file included from /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/uilocale.mm:28:
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/cfstring.h:63:14: note: candidate: 'wxString wxCFStringRef::AsString() const'
   63 |     wxString AsString() const;
      |              ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/cfstring.h:63:14: note:   candidate expects 0 arguments, 1 provided
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/cfstring.h:65:21: note: candidate: 'static wxString wxCFStringRef::AsString(CFStringRef)'
   65 |     static wxString AsString( CFStringRef ref ) ;
      |                     ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/cfstring.h:65:43: note:   no known conversion for argument 1 from 'NSArray' to 'CFStringRef' {aka 'const __CFString*'}
   65 |     static wxString AsString( CFStringRef ref ) ;
      |                               ~~~~~~~~~~~~^~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/cfstring.h:68:21: note: candidate: 'static wxString wxCFStringRef::AsString(WX_NSString)'
   68 |     static wxString AsString( WX_NSString ref ) ;
      |                     ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/cfstring.h:68:43: note:   no known conversion for argument 1 from 'NSArray' to 'WX_NSString' {aka 'NSString*'}
   68 |     static wxString AsString( WX_NSString ref ) ;
      |                               ~~~~~~~~~~~~^~~
In file included from /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/private.h:4,
                 from /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/evtloop_cf.cpp:34:
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/private.h:42:22: warning: "wxOSX_USE_IPHONE" is not defined, evaluates to 0 [-Wundef]
   42 | #if ( !wxUSE_GUI && !wxOSX_USE_IPHONE ) || wxOSX_USE_COCOA_OR_CARBON
      |                      ^~~~~~~~~~~~~~~~
In file included from /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/private.h:4,
                 from /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:24:
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/core/private.h:42:22: warning: "wxOSX_USE_IPHONE" is not defined, evaluates to 0 [-Wundef]
   42 | #if ( !wxUSE_GUI && !wxOSX_USE_IPHONE ) || wxOSX_USE_COCOA_OR_CARBON
      |                      ^~~~~~~~~~~~~~~~
At global scope:
cc1objplus: note: unrecognized command-line option '-Wno-shorten-64-to-32' may have been intended to silence earlier diagnostics
make[2]: *** [libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/uilocale.mm.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/private.h:6:5: warning: "wxOSX_USE_IPHONE" is not defined, evaluates to 0 [-Wundef]
    6 | #if wxOSX_USE_IPHONE
      |     ^~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/private.h:8:7: warning: "wxOSX_USE_COCOA" is not defined, evaluates to 0 [-Wundef]
    8 | #elif wxOSX_USE_COCOA
      |       ^~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/private.h:6:5: warning: "wxOSX_USE_IPHONE" is not defined, evaluates to 0 [-Wundef]
    6 | #if wxOSX_USE_IPHONE
      |     ^~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include/wx/osx/private.h:8:7: warning: "wxOSX_USE_COCOA" is not defined, evaluates to 0 [-Wundef]
    8 | #elif wxOSX_USE_COCOA
      |       ^~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm: In function 'wxOperatingSystemId wxGetOsVersion(int*, int*, int*)':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:43:5: error: 'NSOperatingSystemVersion' was not declared in this scope; did you mean 'kNSLMinSystemVersion'?
   43 |     NSOperatingSystemVersion osVer = [NSProcessInfo processInfo].operatingSystemVersion;
      |     ^~~~~~~~~~~~~~~~~~~~~~~~
      |     kNSLMinSystemVersion
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:46:19: error: 'osVer' was not declared in this scope
   46 |         *verMaj = osVer.majorVersion;
      |                   ^~~~~
cc1plus: note: unrecognized command-line option '-Wno-shorten-64-to-32' may have been intended to silence earlier diagnostics
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:49:19: error: 'osVer' was not declared in this scope
   49 |         *verMin = osVer.minorVersion;
      |                   ^~~~~
cc1plus: note: unrecognized command-line option '-Wno-shorten-64-to-32' may have been intended to silence earlier diagnostics
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:52:21: error: 'osVer' was not declared in this scope
   52 |         *verMicro = osVer.patchVersion;
      |                     ^~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm: In function 'bool wxCheckOsVersion(int, int, int)':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:59:5: error: 'NSOperatingSystemVersion' was not declared in this scope; did you mean 'kNSLMinSystemVersion'?
   59 |     NSOperatingSystemVersion osVer;
      |     ^~~~~~~~~~~~~~~~~~~~~~~~
      |     kNSLMinSystemVersion
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:60:5: error: 'osVer' was not declared in this scope
   60 |     osVer.majorVersion = majorVsn;
      |     ^~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:64:78: warning: 'NSProcessInfo' may not respond to '-isOperatingSystemAtLeastVersion:'
   64 |     return [[NSProcessInfo processInfo] isOperatingSystemAtLeastVersion:osVer] != NO;
      |                                                                              ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:64:78: warning: (messages without a matching method signature will be assumed to return 'id' and accept '...' as arguments)
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:64:80: error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
   64 |     return [[NSProcessInfo processInfo] isOperatingSystemAtLeastVersion:osVer] != NO;
      |                                                                                ^
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm: In function 'bool wxCocoaLaunch(const char* const*, pid_t&)':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/cocoa/utils_base.mm:238:34: warning: 'NSWorkspace' may not respond to '-openURLs:withApplicationAtURL:options:configuration:error:'
  238 |                      error:&error];
      |                                  ^
At global scope:
cc1objplus: note: unrecognized command-line option '-Wno-shorten-64-to-32' may have been intended to silence earlier diagnostics
make[2]: *** [libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/osx/cocoa/utils_base.mm.o] Error 1
make[2]: Leaving directory `/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build'
make[1]: *** [libs/base/CMakeFiles/wxbase.dir/all] Error 2
```

2. Missing `xlocale.h` include in generic uilocale breaks the build: 

```
[ 17%] Building CXX object libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/unix/mimetype.cpp.o
cd /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base && /opt/local/bin/g++-mp-14 -DWXBUILDING -DWXDLLNAME=wx_baseu-3.3 -DWXMAKINGDLL_BASE -DWX_PRECOMP -D_FILE_OFFSET_BITS=64 -D_UNICODE -D__WXGTK3__ -D__WXGTK__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -Dwxbase_EXPORTS -I/opt/local/include -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/lib/wx/include/gtk3-unicode-3.3 -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include -pipe -Os -F/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks -U__APPLE_ALTIVEC__ -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -mmacosx-version-min=10.6 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wundef -Wunused-parameter -Wno-shorten-64-to-32 -Wno-ctor-dtor-privacy -Woverloaded-virtual -Winvalid-pch -include /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/unix/mimetype.cpp.o -MF CMakeFiles/wxbase.dir/__/__/__/__/src/unix/mimetype.cpp.o.d -o CMakeFiles/wxbase.dir/__/__/__/__/src/unix/mimetype.cpp.o -c /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/mimetype.cpp
[ 17%] Building CXX object libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/unix/uilocale.cpp.o
cd /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base && /opt/local/bin/g++-mp-14 -DWXBUILDING -DWXDLLNAME=wx_baseu-3.3 -DWXMAKINGDLL_BASE -DWX_PRECOMP -D_FILE_OFFSET_BITS=64 -D_UNICODE -D__WXGTK3__ -D__WXGTK__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -Dwxbase_EXPORTS -I/opt/local/include -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/lib/wx/include/gtk3-unicode-3.3 -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include -pipe -Os -F/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks -U__APPLE_ALTIVEC__ -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -mmacosx-version-min=10.6 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wundef -Wunused-parameter -Wno-shorten-64-to-32 -Wno-ctor-dtor-privacy -Woverloaded-virtual -Winvalid-pch -include /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/unix/uilocale.cpp.o -MF CMakeFiles/wxbase.dir/__/__/__/__/src/unix/uilocale.cpp.o.d -o CMakeFiles/wxbase.dir/__/__/__/__/src/unix/uilocale.cpp.o -c /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:172:34: error: 'locale_t' has not been declared
  172 |                                , locale_t loc = nullptr
      |                                  ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:222:11: error: 'locale_t' does not name a type
  222 |     const locale_t m_locale;
      |           ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:172:49: error: could not convert 'nullptr' from 'std::nullptr_t' to 'int'
  172 |                                , locale_t loc = nullptr
      |                                                 ^~~~~~~
      |                                                 |
      |                                                 std::nullptr_t
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:231:8: error: 'locale_t' does not name a type
  231 | inline locale_t TryCreateLocale(const wxLocaleIdent& locId)
      |        ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:238:1: error: 'locale_t' does not name a type
  238 | locale_t TryCreateLocaleWithUTF8(wxLocaleIdent& locId)
      | ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:276:1: error: 'locale_t' does not name a type
  276 | locale_t TryCreateMatchingLocale(wxLocaleIdent& locId)
      | ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:398:41: error: 'locale_t' has not been declared
  398 |                                       , locale_t loc
      |                                         ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In constructor '{anonymous}::wxUILocaleImplUnix::wxUILocaleImplUnix(wxLocaleIdent, int)':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:403:21: error: class '{anonymous}::wxUILocaleImplUnix' does not have any field named 'm_locale'
  403 |                   , m_locale(loc ? loc : TryCreateLocale(locId))
      |                     ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:403:42: error: 'TryCreateLocale' was not declared in this scope
  403 |                   , m_locale(loc ? loc : TryCreateLocale(locId))
      |                                          ^~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In destructor 'virtual {anonymous}::wxUILocaleImplUnix::~wxUILocaleImplUnix()':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:411:10: error: 'm_locale' was not declared in this scope
  411 |     if ( m_locale )
      |          ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:412:9: error: 'freelocale' was not declared in this scope; did you mean 'setlocale'?
  412 |         freelocale(m_locale);
      |         ^~~~~~~~~~
      |         setlocale
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In member function 'const char* {anonymous}::wxUILocaleImplUnix::GetLangInfo(nl_item) const':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:515:10: error: 'm_locale' was not declared in this scope
  515 |     if ( m_locale )
      |          ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:516:16: error: 'nl_langinfo_l' was not declared in this scope; did you mean 'nl_langinfo'?
  516 |         return nl_langinfo_l(item, m_locale);
      |                ^~~~~~~~~~~~~
      |                nl_langinfo
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In member function 'const wchar_t* {anonymous}::wxUILocaleImplUnix::GetLangInfoWide(nl_item) const':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:529:10: error: 'm_locale' was not declared in this scope
  529 |     if ( m_locale )
      |          ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:530:27: error: 'nl_langinfo_l' was not declared in this scope; did you mean 'nl_langinfo'?
  530 |         return (wchar_t*) nl_langinfo_l(item, m_locale);
      |                           ^~~~~~~~~~~~~
      |                           nl_langinfo
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In member function 'virtual int {anonymous}::wxUILocaleImplUnix::CompareStrings(const wxString&, const wxString&, int) const':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:837:10: error: 'm_locale' was not declared in this scope
  837 |     if ( m_locale )
      |          ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:838:14: error: 'wcscoll_l' was not declared in this scope; did you mean 'wcscoll'?
  838 |         rc = wcscoll_l(lhs.wc_str(), rhs.wc_str(), m_locale);
      |              ^~~~~~~~~
      |              wcscoll
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In static member function 'static wxUILocaleImpl* wxUILocaleImpl::CreateStdC()':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:855:64: error: cannot convert 'std::nullptr_t' to 'int'
  855 |     return new wxUILocaleImplUnix(wxLocaleIdent().Language("C"));
      |                                                                ^
      |                                                                |
      |                                                                std::nullptr_t
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:398:50: note:   initializing argument 2 of '{anonymous}::wxUILocaleImplUnix::wxUILocaleImplUnix(wxLocaleIdent, int)'
  398 |                                       , locale_t loc
      |                                         ~~~~~~~~~^~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In static member function 'static wxUILocaleImpl* wxUILocaleImpl::CreateUserDefault()':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:866:5: error: 'locale_t' was not declared in this scope
  866 |     locale_t loc = TryCreateLocaleWithUTF8(locDef);
      |     ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:867:11: error: 'loc' was not declared in this scope; did you mean 'log'?
  867 |     if ( !loc )
      |           ^~~
      |           log
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:870:52: error: 'loc' was not declared in this scope; did you mean 'log'?
  870 |     return new wxUILocaleImplUnix(wxLocaleIdent(), loc);
      |                                                    ^~~
      |                                                    log
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: In static member function 'static wxUILocaleImpl* wxUILocaleImpl::CreateForLocale(const wxLocaleIdent&)':
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:885:11: error: 'locale_t' does not name a type
  885 |     const locale_t loc = TryCreateMatchingLocale(locId);
      |           ^~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:886:11: error: 'loc' was not declared in this scope; did you mean 'log'?
  886 |     if ( !loc )
      |           ^~~
      |           log
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:889:42: error: 'loc' was not declared in this scope; did you mean 'log'?
  889 |     return new wxUILocaleImplUnix(locId, loc);
      |                                          ^~~
      |                                          log
[ 17%] Building CXX object libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/cfstring.cpp.o
cd /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base && /opt/local/bin/g++-mp-14 -DWXBUILDING -DWXDLLNAME=wx_baseu-3.3 -DWXMAKINGDLL_BASE -DWX_PRECOMP -D_FILE_OFFSET_BITS=64 -D_UNICODE -D__WXGTK3__ -D__WXGTK__ -DwxUSE_BASE=1 -DwxUSE_GUI=0 -Dwxbase_EXPORTS -I/opt/local/include -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/lib/wx/include/gtk3-unicode-3.3 -I/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/include -pipe -Os -F/System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks -U__APPLE_ALTIVEC__ -DNDEBUG -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -mmacosx-version-min=10.6 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wundef -Wunused-parameter -Wno-shorten-64-to-32 -Wno-ctor-dtor-privacy -Woverloaded-virtual -Winvalid-pch -include /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/build/libs/base/CMakeFiles/wxbase.dir/cmake_pch.hxx -MD -MT libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/cfstring.cpp.o -MF CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/cfstring.cpp.o.d -o CMakeFiles/wxbase.dir/__/__/__/__/src/osx/core/cfstring.cpp.o -c /opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/osx/core/cfstring.cpp
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp: At global scope:
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:525:1: warning: 'const wchar_t* {anonymous}::wxUILocaleImplUnix::GetLangInfoWide(nl_item) const' defined but not used [-Wunused-function]
  525 | wxUILocaleImplUnix::GetLangInfoWide(nl_item item) const
      | ^~~~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:408:1: warning: '{anonymous}::wxUILocaleImplUnix::~wxUILocaleImplUnix()' defined but not used [-Wunused-function]
  408 | wxUILocaleImplUnix::~wxUILocaleImplUnix()
      | ^~~~~~~~~~~~~~~~~~
/opt/local/var/macports/build/_opt_local_ppcports_graphics_wxgtk-devel/wxgtk-devel/work/wxWidgets-5bc736df13144800b64b034399d30a6111ee9cea/src/unix/uilocale.cpp:396:1: warning: '{anonymous}::wxUILocaleImplUnix::wxUILocaleImplUnix(wxLocaleIdent, int)' defined but not used [-Wunused-function]
  396 | wxUILocaleImplUnix::wxUILocaleImplUnix(wxLocaleIdent locId
      | ^~~~~~~~~~~~~~~~~~
```

--- a/build/files
+++ b/build/files
@@ -140,6 +140,7 @@ BASE_COREFOUNDATION_SRC =
     src/osx/core/secretstore.cpp
     src/osx/core/strconv_cf.cpp
     src/osx/core/uilocale.mm
+    src/osx/cocoa/stdpaths.mm
     src/osx/cocoa/utils_base.mm
 
 BASE_COREFOUNDATION_HDR =
@@ -163,7 +164,6 @@ BASE_COREFOUNDATION_HDR =
 BASE_OSX_SHARED_SRC =
     src/osx/core/mimetype.cpp
     src/osx/fswatcher_fsevents.cpp
-    src/osx/cocoa/stdpaths.mm
     $(BASE_COREFOUNDATION_SRC)
     $(BASE_UNIX_AND_DARWIN_SRC)
 

--- a/build/files
+++ b/build/files
@@ -56,6 +56,7 @@ BASE_UNIX_AND_DARWIN_HDR =
 BASE_UNIX_AND_DARWIN_NOTWXMAC_SRC =
     $(BASE_UNIX_AND_DARWIN_SRC)
     src/unix/mimetype.cpp
+    src/unix/uilocale.cpp
 
 BASE_UNIX_AND_DARWIN_NOTWXMAC_HDR =
     $(BASE_UNIX_AND_DARWIN_HDR)
@@ -70,7 +71,6 @@ BASE_UNIX_SRC =
     src/unix/fswatcher_inotify.cpp
     src/unix/secretstore.cpp
     src/unix/stdpaths.cpp
-    src/unix/uilocale.cpp
 
 BASE_UNIX_HDR =
     $(BASE_UNIX_AND_DARWIN_NOTWXMAC_HDR)
@@ -139,7 +139,6 @@ BASE_COREFOUNDATION_SRC =
     src/osx/core/evtloop_cf.cpp
     src/osx/core/secretstore.cpp
     src/osx/core/strconv_cf.cpp
-    src/osx/core/uilocale.mm
     src/osx/cocoa/stdpaths.mm
     src/osx/cocoa/utils_base.mm
 
@@ -163,6 +162,7 @@ BASE_COREFOUNDATION_HDR =
 # Base files used by OS X ports (not Carbon)
 BASE_OSX_SHARED_SRC =
     src/osx/core/mimetype.cpp
+    src/osx/core/uilocale.mm
     src/osx/fswatcher_fsevents.cpp
     $(BASE_COREFOUNDATION_SRC)
     $(BASE_UNIX_AND_DARWIN_SRC)

--- a/src/unix/uilocale.cpp
+++ b/src/unix/uilocale.cpp
@@ -21,6 +21,10 @@
 #if wxUSE_INTL
 
 #include <locale.h>
+#ifdef __APPLE__
+    #include <xlocale.h>
+#endif
+
 #ifdef HAVE_LANGINFO_H
     #include <langinfo.h>
 #endif

